overview/roles.md

```diff
@@ -90,20 +90,20 @@ presentation of translated text on-screen. These are generally called *signs*.
 
 For example, given this scene and a translation of "Adachi Fourth Public High School"…
 
-![\[DameDesuYo\] Eromanga-sensei - 01 (1920x1080 10bit AAC) \[05CB518E\].mkv\_snapshot\_03.11\_\[2017.08.18\_21.14.55\].jpg](images/cnvimage100.png)
+![[DameDesuYo] Eromanga-sensei - 01 (1920x1080 10bit AAC) [05CB518E].mkv_snapshot_03.11_[2017.08.18_21.14.55].jpg](images/cnvimage100.png)
```

Comment:
Don't these underscores cause text to be italicized?

Reply:
Oddly enough, no they don't. It seems the caption plug-in doesn't recognize markdown formatting, and just italicizes everything. (The asterisks are just for an example, and aren't escaped)

![image](https://user-images.githubusercontent.com/10972040/61573640-a2d78280-aaa1-11e9-950c-334226c7c887.png)

Reply:
This is on the descaling page for example, no formatting.

![image](https://user-images.githubusercontent.com/10972040/61573662-ee8a2c00-aaa1-11e9-8f82-ca112014463e.png)

Reply:
Thanks for checking.